packages/zwave-js/src/lib/driver/UpdateConfig.ts

```diff
@@ -64,11 +65,38 @@ export async function installConfigUpdate(newVersion: string): Promise<void> {
 		);
 	}
 
+	const pkgFilepath = `${pak.cwd}/package.json`;
+
+	await lockfile.lock(pkgFilepath, {
+		// We cannot be sure that the file exists before acquiring the lock
+		realpath: false,
+
+		stale:
+			// Avoid timeouts during testing
+			process.env.NODE_ENV === "test"
+				? 100000
+				: /* istanbul ignore next - this is impossible to test */ undefined,
+
+		onCompromised: /* istanbul ignore next */ () => {
+			// do nothing
+		},
+	});
+
 	// And install it
 	const result = await pak.overrideDependencies({
 		"@zwave-js/config": newVersion,
 	});
+
+	// Free the lock
+	try {
+		if (await lockfile.check(pkgFilepath, { realpath: false }))
+			await lockfile.unlock(pkgFilepath, { realpath: false });
```

Comment:
See comment above regarding realpath
```suggestion
		if (await lockfile.check(pkgFilepath))
			await lockfile.unlock(pkgFilepath);
```